domains/ibradecode.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ibradecode",
+        "email": "ibradecode@gmail.com"
+    },
+    "records": {
+        "CNAME": ["979ea37a9c105372.vercel-dns-017.com."]
```

```suggestion
        "CNAME": "979ea37a9c105372.vercel-dns-017.com"
```

CNAME records is an instance of a string